index.html

```diff
@@ -134,14 +134,14 @@ <h4 class="project-title">CC ignite</h4>
 
     <!-- partners -->
     <section class="partners">
-      <h3 class="p-heading">Partner</h3>
+      <h3 class="d-access">Partner</h3>
       <hr />
       <div class="partners-container">
-        <img src="assets/images/partners/mozilla.png" alt="" class="partner" />
-        <img src="assets/images/partners/google.png" alt="" class="partner" />
-        <img src="assets/images/partners/naver.png" alt="" class="partner" />
-        <img src="assets/images/partners/daumkakao.png" alt="" class="partner" />
-        <img src="assets/images/partners/bnb.png" alt="" class="partner" />
+        <img src="assets/images/partners/mozilla 1.png" alt="" class="access" />
+        <img src="assets/images/partners/googlekorea 1.png" alt="" class="access" />
+        <img src="assets/images/partners/naver 1.png" alt="" class="access" />
+        <img src="assets/images/partners/daumkakaologo 1.png" alt="" class="access" />
+        <img src="assets/images/partners/airbnb 1.png" alt="" class="access" />
```

Comment:
Great design @maths29 🤝

- [ ] For semantic purposes I would recommend you add `alt` attributes to all your images across your HTML files and assign them values.